.github/workflows/tests-registry.yml

```
@@ -0,0 +1,56 @@
+  pull_request:
+    branches: [main]
+    paths:
+      - 'platforms/registry/**'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)